server/vulnerabilities/sync.go

```diff
@@ -202,6 +202,7 @@ func LoadCVEMeta(vulnPath string, ds fleet.Datastore) error {
 	}
 
 	for _, epssScore := range epssScores {
+		epssScore := epssScore // copy, don't take the address of loop variables
```

Comment:
Would it be more idiomatic to use a `for i := range epssScores { .. } ...` here instead?

Reply:
I'm not sure this is a question of what's more idiomatic, rather more of a stylistic choice.